include/BoostInstall.cmake

```diff
@@ -315,6 +315,12 @@ function(boost_install_target)
         string(APPEND CONFIG_FILE_CONTENTS "  find_dependency(boost_${CMAKE_MATCH_1} ${__VERSION} EXACT)\n")
         string(APPEND CONFIG_FILE_CONTENTS "endif()\n")
 
+      elseif(dep MATCHES "^\\\$<TARGET_NAME_IF_EXISTS:Boost::(.*)>$")
```

Comment:
Are the three slashes here correct? Shouldn't there be just two?

Reply:
Tested it. Both work. For some reason I made a habit out of using 3.

Reply:
I think that the third slash just quotes the $ unnecessarily.

Can you please remove it?